The GitHub archive we use doesn't contain all the info necessary to guess the
package version automatically, which is why it needs to be listed explicitly here.

Not intended for upstream.

--- iniconfig-2.1.0/pyproject.toml
+++ iniconfig-2.1.0/pyproject.toml
@@ -7,6 +7,7 @@ requires = [
 
 [project]
 name = "iniconfig"
+version = "2.1.0"
 description = "brain-dead simple config-ini parsing"
 readme = "README.rst"
 license = "MIT"
@@ -15,9 +16,6 @@ authors = [
     { name = "Holger Krekel", email = "holger.krekel@gmail.com" },
 ]
 requires-python = ">=3.8"
-dynamic = [
-  "version",
-]
 classifiers = [
     "Development Status :: 4 - Beta",
     "Intended Audience :: Developers",
